docs/kata-containers.md

```diff
@@ -0,0 +1,44 @@
+# Kata containers support in CRI-RM
+
+[Kata Containers](https://katacontainers.io/) is an open source container runtime, 
+building lightweight virtual machines that seamlessly plug into the containers ecosystem.
+
+In order to enable Kata container in a Kubernetes-CRI-RM stack, both Kubernetes 
+and the Container Runtime Engine need to be aware of the new runtime environment:
+
+  * The Container Runtime Engine can only be CRI-O or containerd, and will need to
+   have the runtimes enabled in their configuration files.
```

Comment:
I think the terminology is a bit misleading here. I consider *container runtime engine* to be the lower level OCI compliant runtime like runc or kata. The upper level compoment (like containerd or CRI-O) would be *container runtime* or *container engine*. Probably *container runtime* in our context. 

What do others think?

Reply:
Okay. I thought it would make sense since we are also referencing Kata containers & Runc as being different runtimes for containers, which might create confusion if containerd and CRI-O become container runtimes themselves.